scripts/treedb_vectordbbench_artifact.py

```
@@ -899,6 +909,8 @@ def parse_args(argv: list[str]) -> argparse.Namespace:
     args = parser.parse_args(argv)
     try:
         validate_smoke_shape(args.smoke_dimension, args.smoke_documents, args.smoke_top_k, args.ef_search, args.rerank_candidates)
+        if args.num_per_batch <= 0:
+            raise ValueError("num-per-batch must be positive")
```

Ruff (0.16.1)

[warning] Abstract `raise` to an inner function

(TRY301)